pkg/services/alerting/store.go

```diff
@@ -107,8 +108,13 @@ func deleteAlertByIdInternal(alertId int64, reason string, sess *db.Session, log
 }
 
 func (ss *sqlStore) HandleAlertsQuery(ctx context.Context, query *alertmodels.GetAlertsQuery) (res []*alertmodels.AlertListItemDTO, err error) {
+	recursiveQueriesAreSupported, err := ss.db.RecursiveQueriesAreSupported()
+	if err != nil {
+		return res, err
+	}
+
 	err = ss.db.WithDbSession(ctx, func(sess *db.Session) error {
-		builder := db.NewSqlBuilder(ss.cfg, ss.db.GetDialect())
+		builder := db.NewSqlBuilder(ss.cfg, featuremgmt.WithFeatures(), ss.db.GetDialect(), recursiveQueriesAreSupported)
```

Comment:
question: would it be better to pass features in properly? I think `featuremgmt.WithFeatures()` is mainly used for testing, and could cause confusion if we rely on it in SQL builder.